packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/TransactionsCompletedRow.tsx

```diff
@@ -16,7 +16,7 @@ const TransactionsCompletedRow: React.FC<TProps> = ({ accounts, transaction, wal
     const dxtradeOrCtraderToFrom = useMemo(() => {
         if (transaction?.action_type !== 'transfer' || !transaction.longcode) return null;
         const longcodeMessageTokens = transaction.longcode.split(' ');
-        const direction = longcodeMessageTokens[1];
+        const direction = longcodeMessageTokens[4] === 'cTrader' ? 'to' : longcodeMessageTokens[1];
```

Comment:
approving, as this pattern was already there as its marked as "TODO: remove it", but for future, we really shouln't need to "guess" the direction like that, never should hardcode indexes to select individual bits of some "long code"

Reply:
agreed 👍 but as stated this is only a temporary workaround before backend adds the to and from fields for Deriv X and CTrader transfers

Reply:
Agreed. There's a BE ticket to address this which is in progress, we **should** come back to this when it's released.